### PR TITLE
[FR] Localize `Continuous Integration`

### PR DESCRIPTION
**Pull request by guillaumebernard84**
### Describe your changes
Localize `Continuous Integration` to french

### Related issue number or link (ex: `resolves #issue-number`)
solves https://github.com/cncf/glossary/issues/2292

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/continuous-integration.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+L'intégration continue, souvent abrégée en CI, est la pratique consistant à intégrer les modifications de code aussi régulièrement que possible.
+La CI est un prérequis à la [livraison continue](/fr/continuous-delivery/) (CD).
+Traditionnellement, le processus CI commence lorsque les modifications du code sont envoyées dans un système de contrôle de code source (Git, Mercurial ou Subversion).
+et se termine par un artefact testé prêt à être consommé par un système CD.
+
+## Problème auquel il répond
+
+Les systèmes logiciels sont souvent grands et complexes, et de nombreux développeurs les maintiennent et les mettent à jour.
+En travaillant en parallèle sur différentes parties du système,
+ces développeurs peuvent apporter des modifications contradictoires et, par inadvertance, se causer les uns les autres des problémes.
```
Comment (sestegra) [inline content/fr/continuous-integration.md:19]: Typo "problèmes"

File: content/fr/continuous-integration.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+status: Completed 
+category: concept
+tags: ["application", "méthodologie", ""]
+---
+
+## Ce que c'est
+
+L'intégration continue, souvent abrégée en CI, est la pratique consistant à intégrer les modifications de code aussi régulièrement que possible.
+La CI est un prérequis à la [livraison continue](/fr/continuous-delivery/) (CD).
+Traditionnellement, le processus CI commence lorsque les modifications du code sont envoyées dans un système de contrôle de code source (Git, Mercurial ou Subversion).
```
Comment (sestegra) [inline content/fr/continuous-integration.md:12]: Je propose "de code" au lieu "du code" qui est plus général.

Comment (sestegra) [inline content/fr/continuous-integration.md:12]: Typo: une virgule à la place du point "(Git, Mercurial ou Subversion),"

File: content/fr/continuous-integration.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Les systèmes logiciels sont souvent grands et complexes, et de nombreux développeurs les maintiennent et les mettent à jour.
+En travaillant en parallèle sur différentes parties du système,
+ces développeurs peuvent apporter des modifications contradictoires et, par inadvertance, se causer les uns les autres des problémes.
+De plus, avec plusieurs développeurs travaillant sur le même projet,
+toutes les tâches quotidiennes telles que tester ou mesurer la qualité du code devraient être répétées par chaque développeur, ce qui ferait perdre du temps.
+
+## Quel en est l'utilité
+
+Le logiciel CI vérifie automatiquement que les modifications de code sont fusionées proprement chaque fois qu'un développeur soumet une modification.
```
Comment (sestegra) [inline content/fr/continuous-integration.md:25]: Je propose "Le logiciel de CI"

**Review by sestegra — CHANGES_REQUESTED** [review]

File: content/fr/continuous-integration.md (hunk 2 above)
Comment (guillaumebernard84) [reply content/fr/continuous-integration.md:12]: Tu as raison, c'est mieux

**Review by sestegra — APPROVED** [review]
LGTM 🚀

**Review by Krast76 — APPROVED** [review]
LGTM 🚀

**Pull request merged**